tibanna_ffcommon/portal_utils.py

```diff
@@ -1260,6 +1260,18 @@ def update_input_extras(self):
                     self.update_patch_items(ip['uuid'], {'higlass_uid': higlass_uid})
             self.update_patch_items(ip['uuid'], {'extra_files': ip['extra_files']})
 
+    def get_portal_specific_item_name(self, item: str) -> str:
+        """This function gets the portal specific item names for a given item name. This is necessary because
+        the nameing of similar items can vary across portals, e.g. "quality_metric_generic" (CGAP) vs 
+        "quality_metric" (SMaHT)
```

Comment:
Is this update needed for `pony` as well?

Reply:
In pony we never reach `update_generic_qc` which is the only function that is using the mapping. I think we can add it to pony, when we actually need it.